signac/__main__.py

```diff
@@ -185,7 +185,7 @@ def _read_index(project, fn_index=None):
     if fn_index is not None:
         _print_err("Reading index from file '{}'...".format(fn_index))
         fd = open(fn_index)
-        return (json.loads(l) for l in fd)
+        return (json.loads(i) for i in fd)
```

Comment:
```suggestion
        return (json.loads(_) for _ in fd)
```

Reply:
Or even better, a descriptive name like `line`.
```suggestion
        return (json.loads(line) for line in fd)
```